src/FileLogger.java

```diff
@@ -0,0 +1,57 @@
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class FileLogger {
+    private FileLoggerConfiguration fileLoggerConfiguration;
+    private String configurationFile;
+
+    public FileLogger(String configurationFileName) {
+        this.configurationFile = configurationFileName;
+        this.fileLoggerConfiguration = new FileLoggerConfigurationLoader().load(configurationFileName);
+    }
+
+    public void log(String message) {
+        String text = LocalTime.now() + " " + fileLoggerConfiguration.getLoggingLevel() + " Повідемлення: " + message;
+        info(text);
+        if (fileLoggerConfiguration.getLoggingLevel().equals(LoggingLevel.DEBUG)) {
+            try {
+                debug(text);
+            } catch (FileMaxSizeReachedException fileMaxSizeReachedException) {
+                File file = new File("Log_"
+                        + LocalDateTime.now().format(DateTimeFormatter.ofPattern("dd.MM.yyyy-HH.mm"))
+                        + "."
+                        + fileLoggerConfiguration.getFormat());
+                try {
+                    file.createNewFile();
+                    fileLoggerConfiguration.setFile(file);
+                    new FileLoggerConfigurationLoader().write(fileLoggerConfiguration, configurationFile);
+                    debug(text);
+                } catch (IOException e) {
+                    throw new RuntimeException(e);
+                }
+            }
+        }
+    }
+
+    private void debug(String message) {
```

Comment:
This method should call `log()` from here like this: 
`if (isDebugEnabled()) {
     log(message, LoggingLevel.DEBUG)
 }`
, do not use a print statement for that.